CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# TensorFlow Addons Codeowners
+
+# SIG maintainers are the catch all for reviews
+* @tensorflow/sig-addons-maintainers
+
+
+# Subpackage Owners
+seq2seq/* @qlzh727
+
+
+# Submodule Owners
+# These may not automatically trigger a review because submodule owners do not
+# always have write access. But still adding these here for
+# repository understanding.
+
+sparsemax* @AndreasMadsen
```

Comment:
`/tensorflow_addons/activations/sparsemax*` ?

Reply:
Same reason... Sparsemax also located in activations, layers, and losses